app/services/person_constructor.rb

```diff
@@ -0,0 +1,82 @@
+class PersonConstructor
+  KEY_NAME_MAPPINGS = {
+    address: :address_1,
+    zip: :zip_code,
+  }
+  OLD_REMOTE_PARAMS = [
+    remote_fields: PersonWithRemoteFields::REMOTE_PARAMS
+  ]
+
+  def self.permitted_params
```

Comment:
I realize you're adding many things here, but I'd just make this a constant and refer to it as `PersonContructor::PermittedParams`